### PR TITLE
Extracted properties support

### PR DESCRIPTION
**Pull request by zerodelayy**
Adding support for element extracted properties.

### CODE REVIEW

**Comment by zerodelayy** [comment]
Also added support for element extracted properties.

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ This dbt package contains models, macros, seeds, and tests for [Fullstory](https
 | fullstory_event_types | A list of event types to auto-generate rollups for in the `users` and `sessions` model. |
 
 > We **highly recommend** using `fullstory_events_database`, `fullstory_events_schema` and `fullstory_events_table` to indicate the location of the Fullstory events table that is synced from Data Destinations. Using these variables allow you to use a separate database or schema for the Fullstory events table than your dbt package.
+>
+> We **highly recommend** utilizing the Fullstory OrgID as the `target` name in `profiles.yml`. This will not only make Umbrella configurations easier but will also allow for `replay_url.sql` to function as intended.
```
Comment (camphillips22) [inline README.md:36]: Let's remove this from this PR for the moment. The placement of this guidance is a bit off given it's in the `Vars` section. I don't want to block the new field update on getting this right, however.

File: integration_tests/models/_test_events.yml [hunk 2]
```diff
@@ -208,3 +208,13 @@ models:
           expression: "{{ dbt_fullstory.json_value('event_properties', '$.user_display_name', skip_parse=True) }} = 'good_display_name'"
           config:
             where: event_id = 'user_total_sessions_test_3'
+      - dbt_utils.expression_is_true:
+          name: element_properties_testPrice_is_correct
+          expression: "{{ dbt_fullstory.json_value('event_properties', '$.target.event_properties.testPrice', skip_parse=True) }} = 3.14"
```
Comment (camphillips22) [inline integration_tests/models/_test_events.yml:213]: Shouldn't this be `target.element_properties` not `target.event_properties`?
```suggestion
          expression: "{{ dbt_fullstory.json_value('event_properties', '$.target.element_properties.testPrice', skip_parse=True) }} = 3.14"
```

File: README.md (hunk 1 above)
Comment (zerodelayy) [reply README.md]: That should be undone now, thanks for pointing it out!

File: integration_tests/models/_test_events.yml (hunk 2 above)
Comment (zerodelayy) [reply integration_tests/models/_test_events.yml:213]: ah very good catch, thank you!

**Comment by camphillips22** [comment]
Also, since we remove the readme update, can you update the PR and description?

**Pull request closed without merging**